packages/trader/src/sass/app/_common/components/contract-type-info.scss

```diff
@@ -197,4 +197,14 @@
             }
         }
     }
+    &__video {
+        width: 100%;
```

Comment:
does it work with width being 480px instead of 100%?

Reply:
Yes, you can try test link

Reply:
ok lets change it to 480px or 48rem. That width is per design i believe

Reply:
Sorry, my bad, it'll work with 100%, but not with rem/px.